zebedee-cms/src/main/java/com/github/onsdigital/zebedee/json/CollectionDataset.java

```diff
@@ -42,6 +43,14 @@ public void setUri(String uri) {
         this.uri = uri;
     }
 
+    public void setLastEditedBy(String user) {
+        this.lastEditedBy = user;
+    }
+
+    public String getLastEditedBy() {
+        return lastEditedBy;
+    }
+
     @Override
     public boolean equals(Object o) {
         if (this == o) return true;
```

Comment:
I dont know what the equals method is used for in the code base but if you have added a new field to this class does that change the definition of equality? Basically do we want to factor `lastEditedBy`into deciding if 2 CollectionDatasets are equal.
If you do update the equals method you should also update hashCode to take into account the new field.

Reply:
Thanks, I hadn't considered that. I think considering we don't seem to mind about the other fields for equality (i.e. it's only dependent on the ID being the same) I assume the `lastEditedBy` field shouldn't either.